src/main/java/mate/jdbc/service/impl/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.ManufacturerService;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+
+@Service
+public class ManufacturerServiceImpl implements mate.jdbc.service.ManufacturerService {
+    private static final String GET_MANUFACTURER_BY_ID_EXCEPTION =
+            "Can`t get Manufacturer by id - %s";
+
+    @Inject
+    private ManufacturerService manufacturerService;
```

Comment:
ManufacturerServices that is using ManufacturerService as dependency is awful :)